src/main/kotlin/minesweeper/domain/TileRow.kt

```diff
@@ -20,7 +20,27 @@ class TileRow(private val row: MutableList<Tile>) {
         row[position] = tile
     }
 
-    fun filter(): List<Tile> {
+    fun filterMine(): List<Tile> {
         return row.filter { it.isMine() }
     }
+
+    fun sizeOfUnChecked(): Int {
+        return row.filter { !it.isCheckedTile }.size
+    }
+
+    fun inCreaseMineCountNearTile(rowRange: List<Int>) {
+        for(rowIndex in rowRange) {
```

Comment:
아래가 조금 더 짧은데 아래처럼 사용해보셔도 좋을 것 같네요~

```suggestion
    rowRange.forEach {
```

Reply:
짧다고 말씀하시는게 코드 양 말씀하시는거겠죠?

Reply:
네 맞습니다. 코드 길이도 짧아지고 코틀린에서 DSL 형태({}를 사용하는 형태)로 코드를 많이 작성하는데 for 구문보다는 forEach가 좀 더 코틀린의 전체적인 코드 작성 방식과 유사한 것 같아서 제안드렸습니다.